### ISSUE TITLE
the loss of pythia training

### ISSUE THREAD

**Issue by Wangpeiyi9979** [EleutherAI/pythia#97]
Hi, Is there any information about the loss of Pythia pre-training process like LLaMA?
<img width="548" alt="image" src="https://user-images.githubusercontent.com/42565075/235028606-33ee9834-190c-482b-bc16-9d82f89c16cb.png">

**Comment by haileyschoelkopf** [comment]
https://wandb.ai/eleutherai/pythia?workspace=user-schoelkopf

Hi! We have a public wandb board (linked above) for our training runs. if you filter this by runs without "crashed" status and runs that lasted longer than 1 hour, then runs named "v2-MODELSIZE" should be what you want (I can help point out specific runs if needed, it is on my todo list to clean this up.)

**Comment by Wangpeiyi9979** [comment]
Thanks

**Comment by xiaoda99** [comment]
Hi, I can only find runs for 160M model in the above wandb link. And loading the many pages is very slow. Could you provide a cleaned versoin of runs for v2 6.9B and 12B (withou dedup)? I'm doing experiments on some new transformer architectures and want to compare my training loss results with Pythia's
